Sources/SwiftKueryORM/Model.swift

```diff
@@ -919,6 +919,25 @@ public extension Model {
     return orderByArray
   }
 
+  /**
+    This function constructs an array of OrderBy from the Order values
+  */
+  private static func getOrderBy(order: [Order], table: Table) throws -> [OrderBy] {
+    var orderByArray: [OrderBy] = []
+    for order in order {
+      guard let column = table.columns.first(where: {$0.name == order.value}) else {
+        throw RequestError(.ormInternalError, reason: "Column \(order.value) not found")
```

Comment:
Instead of throwing and `.ormInternalError`, we should throw `.ormQueryError` because it failed during the construction of the Query?